README.md

```diff
@@ -312,3 +312,62 @@ reported properly.
+
+## Creating a fork on Heroku
+
+If you want to run your own add-on based on this codebase, you should follow these instructions. These instructions are for deploying your add-on to Heroku, which is not required. You may have to modify the deploy instructions for other environments.
+
+To start, you may wish to fork the codebase to your own private GitHub repo, so that you can make changes for your specific add-on's use case.
+
+Create a Heroku app. Typically, we name the Heroku app after the slug (or commandline identifier) for the add-on. If I was creating an add-on with the slug of `sudo-sandwich`, I'd probably name the Heroku app `sudo-sandwich` as well. This is not required; and you can name it whatever you'd like. Please note that `sudo-sandwich` is already taken, and slugs are both immutable and must be globally unique!
+
+```
+heroku apps:create sudo-sandwich
+```
+
+You'll need a Heroku Postgresql database to store data in:
+
+```
+heroku addons:create heroku-postgresql --app sudo-sandwich
+```
+
+**TODO** Instructions for deploy the code to Heroku go here.
+
+Generate a manifest with the [addons-admin](https://github.com/heroku/heroku-cli-addons-admin) CLI plugin. See the linked repo for install instructions. After install, run `heroku addons:admin:manifest:generate` and follow the prompts.
+
+
+Allow the `addons-admin` CLI plugin to generate a secret and SSO salt for you. It will save a new file called `addon-manifest.json`. You may need to edit this file to suit your local dev environment's port.
+
+Set secrets from the `addon-manifest.json` on your newly-created app under the following config vars:
+
+```
+heroku config:set SLUG=<slug-from-manifest> PASSWORD=<password-from-manifest> SSO_SALT=<salt-from-manifest> --app sudo-sandwich
+```
+
+You'll also have to generate an encryption key for the database to store secrets with. In a Ruby terminal (pry or irb), run `require 'securerandom'; SecureRandom.hex(32)` to generate an encryption key of the appropriate length. Then set that key on the Heroku app:
+
+```
+heroku config:set ENCRYPTION_KEY=<value-from-securerandom> --app sudo-sandwich
+```
+
+Once you've completed the above, you should be ready to push the manifest to the Heroku API:
+
+```
+heroku addons:admin:manifest:push
+```
+
+If that doesn't work, you'll need to sign up at the [Partner Portal](https://addons-next.heroku.com) as an Add-on Partner.
+
+You will also want to scale the `worker` dyno up so that async provisioning can be handled in the background:
+
+```
+heroku ps:scale worker=1 --app sudo-sandwich
+```
+
+You will need to go to the [Partner Portal](https://addons-next.heroku.com) and add [plans](https://github.com/heroku/sudo-sandwich/blob/master/app/models/sandwich.rb#L2-L8) to your add-on based on the hardcoded values in the `Sandwich` class. You may wish to modify these, for example, to create a plan called `async` for testing async provisioning.
```

Comment:
Maybe add a caveat here: if someone wants to use the same plans, the will need to do this. They just need to make sure that the pans in the partner portal match the `Sandwich` plans